core/drivers/clk/clk-stm32mp13.c

```diff
@@ -1517,17 +1517,17 @@ static int clk_stm32_load_vco_config_fdt(const void *fdt, int subnode,
 	vco->status = RCC_PLLNCR_DIVPEN | RCC_PLLNCR_DIVQEN |
 		      RCC_PLLNCR_DIVREN | RCC_PLLNCR_PLLON;
 
-	vco->frac = _fdt_read_uint32_default(fdt, subnode, "frac", 0);
+	vco->frac = fdt_read_uint32_default(fdt, subnode, "frac", 0);
 
-	vco->src = _fdt_read_uint32_default(fdt, subnode, "src", UINT32_MAX);
+	vco->src = fdt_read_uint32_default(fdt, subnode, "src", UINT32_MAX);
 
 	return 0;
 }
 
 static int clk_stm32_load_output_config_fdt(const void *fdt, int subnode,
 					    struct stm32_pll_output *output)
 {
-	return _fdt_read_uint32_array(fdt, subnode, "st,pll_div_pqr",
+	return fdt_read_uint32_array(fdt, subnode, "st,pll_div_pqr",
```

Comment:
indentation below.

Reply:
Ok, I still apply your tag on the patch and make the fixup